calabash/Classes/Utils/LPDevice.h

```diff
@@ -39,8 +39,11 @@ extern NSString *const LPDeviceSimKeyVersionInfo;
 - (BOOL) isIPhone4Like;
 - (BOOL) isIPhone5Like;
 - (BOOL) isLetterBox;
-- (BOOL) isIPhone10Like;
-- (BOOL) isIPhone10LetterBox;
+- (BOOL) isIPhoneXLike;
+- (BOOL) isIPhoneXSLike;
+- (BOOL) isIPhoneXSMaxLike;
+- (BOOL) isIPhoneXRLike;
+- (BOOL) isIPhoneXLetterBox;
```

Comment:
I want to use `isIPhone10Like` instead of `isIPhoneXLike` - the X here is confusing.  The spoken name of the phone is "iPhone ten".

We don't need `isIPhone10SLike` because the 10s has the same resolution as the 10.  The 10S Max and 10R have new resolutions.

Reply:
*Changed.*